script/export_for_foi/export_hosts.sql

```diff
@@ -0,0 +1,6 @@
+COPY (
+  SELECT sites.abbr, hostname
```

Comment:
Let's do `sites.abbr AS "Site abbreviation"` in all three cases, to be more explicit.